index.html

```diff
@@ -179,6 +186,7 @@ <h3>Working Group Notes, registries etc</h3>
       <ul>
         <li><a href="https://github.com/w3c/imsc-vnext-reqs/pulse">IMSC vNext Requirements</a></li>
         <li><a href="https://github.com/w3c/png-hdr-pq/pulse">PQ HDR in PNG</a></li>
+        <li><a href="https://github.com/w3c/tt-module-registry/pulse">TTML  Extensions Module Registry</a></li>
```

Comment:
Remove this since we have removed the concept of a module registry (for the time being). We can always add it again in the future if we decide to resurrect the concept.

Reply:
Removed.